codex_of_the_damned/navigation.py

```diff
@@ -138,6 +138,60 @@ def walk(self, path=None, top=None, ante=None, post=None):
                 Nav(lazy_gettext("Weenie AUS")),
                 Nav(lazy_gettext("Weenie DEM")),
                 Nav(lazy_gettext("Weenie DOM")),
+                Nav(
+                    lazy_gettext("Bibliodèque"),
+                    index=True,
```

Comment:
I would remove this `index=True` and put the content of `archetypes/bibliodeque/index.html` directly into `archetypes/index.html`, this would leave us with less pages, which I think is always good

Reply:
wip

Reply:
resolved